docs/paper.md

```diff
@@ -80,7 +80,7 @@ ASTE covers a comprehensive, flexible and resuable toolchain for development, te
 # Functionality & Use
 
 The central interface of ASTE is given through a VTK mesh file, which contains information about the geometric shape of the model we emulate.
-The VTK files can be generated from mesh generation tools (e.g., GMSH [@gmsh]), [Python scripts](https://github.com/precice/aste/tree/develop/tools/mesh-generators) or directly reused from a [completed preCICE simulation](https://precice.org/configuration-export.html).
```

Comment:
Some simulation software can export to VTK.
This may be an important first step for evaluating mapping performance before deciding to using preCICE.